example_4x_adj_vbar_pak.yaml

```diff
@@ -0,0 +1,504 @@
+# Example configuration for four ADJ VBar Pak fixtures daisy chained with XLR cables
+# DMX description: https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/8659/vbar_pak.pdf
+# Fixture configuration set for DMX 8 channel mode
+
+uart:
+  id: uart_bus
+  baud_rate: 250000
+  tx_pin: 2 #ESP8266
+  stop_bits: 2
+
+dmx512:
+  id: dmx
+  uart_id: uart_bus
+  tx_pin: 2 #ESP8266
+  uart_num: 1
+  force_full_frames: true
```

Comment:
That's the default, remove this line.

Reply:
added that info to the readme

Reply:
Sorry, it's not the default (I fixed that), but I do not recommend it.